### PR TITLE
raiseorlaunch: init at 2.3.3

### PR DESCRIPTION
**Pull request by winpat**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/applications/misc/raiseorlaunch/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+python3Packages.buildPythonApplication rec {
+  pname = "raiseorlaunch";
+  version = "2.3.3";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "3d694015d020a888b42564d56559213b94981ca2b32b952a49b2de4d029d2e59";
+  };
+
+  doCheck = false;
```
[the remark this answers is not in the document]
Comment (jonringer) [reply pkgs/applications/misc/raiseorlaunch/default.nix]: if the pypi package doesn't include tests, please pull from github.

Tests are really important in maintaining a package, and verifying that it's still correct even when upstream packages get bumped.

**Review by jonringer — COMMENTED** [review]
otherwise LGTM

File: pkgs/applications/misc/raiseorlaunch/default.nix (hunk 1 above)
[the remark this answers is not in the document]
Comment (winpat) [reply pkgs/applications/misc/raiseorlaunch/default.nix]: Thank you for your feedback. I will try to switch over to GitHub in the next couple of days.

File: pkgs/applications/misc/raiseorlaunch/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  buildInputs = [ python3Packages.setuptools_scm ];
```
[the remark this answers is not in the document]
Comment (winpat) [reply pkgs/applications/misc/raiseorlaunch/default.nix]: Fixed.

File: pkgs/applications/misc/raiseorlaunch/default.nix (hunk 1 above)
[the remark this answers is not in the document]
Comment (winpat) [reply pkgs/applications/misc/raiseorlaunch/default.nix]: I removed `doCheck = false` and added the required checkInputs.

**Review by Lassulus — APPROVED** [review]
tested with nix-review

**Pull request merged**